components/CovidMap.js

```diff
@@ -319,21 +316,23 @@ class CovidMap extends React.Component {
       thirdLegendText: '',
       infowindow: null,
     };
+    this.updateData = this.updateData.bind(this);
   }
 
-  componentDidMount() {
+  componentWillMount() {
     this.getGoogleMaps();
+  }
 
-    const { googleSheetsKey, googleSheetsName } = this.state;
+  componentDidMount() {
     fetchAPI();
-    Tabletop.init({
-      key: googleSheetsKey,
-      wanted: [googleSheetsName],
-      callback: googleData => {
-        this.setState({ data: googleData });
-      },
-      simpleSheet: true,
-    });
+    Papa.parse(
+      'https://docs.google.com/spreadsheets/d/e/2PACX-1vRXnMNfqFc9YHOLG4K9WIXNZAcaZY-YSqo_h3q49JIaaN1FM_O6bGXcbhv2TDOUPV6cMHFn9zCt68dM/pub?gid=0&single=true&output=csv',
```

Comment:
It looks like this is a public document, but I just want to confirm we are ok storing this url publicly in our repo.

Reply:
Yeah, it’s a public spreadsheet published on the web. I suppose if we wanted to we could store the sheet ID in Netlify.

Reply:
No need, I think as long as we intend it to be public there is no harm.